graylog2-web-interface/src/components/common/TimeUnitInput.jsx

```diff
@@ -212,7 +213,7 @@ const TimeUnitInput = createReactClass({
           <InputGroup>
             {(!required && !hideCheckbox) && checkbox}
             <FormControl type="number" disabled={!this._isChecked()} onChange={this._onUpdate} value={lodash.defaultTo(this._getEffectiveValue(), '')} />
-            <DropdownButton componentClass={InputGroup.Button}
+            <DropdownButton componentClass={InputGroup.Button.type}
```

Comment:
Looking at the example in the documentation and other components not using the `.type` (`JSONValueInput` and `TimeUnitTimeHistogramPivot` in the server), it looks like leaving it out still works. Is adding the `.type` necessary?